### DISCUSSION TITLE
Lidl HG08010 wont join network

### DISCUSSION THREAD

**Discussion by denstu** [Koenkk/zigbee2mqtt#23760] [Devices]
Hello.

I followed the instructions on https://www.zigbee2mqtt.io/devices/HG08010.html, but HG08010 wont join the network. 
I activated learing mode in zigbee, put device in paring mode - nothing. Tried a few times. No info in log. Reboot of Z2M - no change. Other devices work fine. 

Zigbee 1.39.1
Coordinator-Typ zStack3x0
Coordinator-Version 20240710
Coordinator is a SONOFF Zigbee 3.0 USB Dongle Plus

Ideas?

**Comment by radimsejk** [comment]
I have a similar problem. I accidentally started pairing mode on my HG08010. I activated "join all" in Z2M, the light is in pairing mode (pulsating white light), but it won't join again.

1.36.0-dev commit: 720fc491
EZSP v12
7.3.1.0 build 176
SkyConnect
